### ISSUE TITLE
Change to column_limit does not seem to make a difference

### ISSUE THREAD

**Issue by kevinlondon** [google/yapf#64]
I'm working in a Django codebase and we have an limit on the number of characters at 99 in a line. Following the example and the code, I did the following call:

`$ PYTHONPATH=yapf python yapf/yapf -d -r myprojects/tests/ --style='{based_on_style: pep8, column_limit: 99}'`

YAPF chooses to reformat lines that are just a hair above 80 characters, even with the above settings. For example:

```
-            self.assert_asset_in_filtered_qs(huge_asset, greater_than, value=9000)
+            self.assert_asset_in_filtered_qs(huge_asset, greater_than,
+                                             value=9000)
```

The original line is 82 characters long so I would want to leave it alone. Am I setting the option incorrectly?

**Comment by gwelymernans** [comment]
Hi Kevin,

I'm not able to replicate this. As a sanity check, are you using top-of-tree?

```
$  PYTHONPATH=yapf python yapf/yapf --style='{based_on_style: pep8, column_limit: 99}' /tmp/a.py
class a:
    def foo():
        if True:
            self.assert_asset_in_filtered_qs(huge_asset, greater_than, value=9000)
```

**Comment by kevinlondon** [comment]
Hi, 

I figured out what it is, actually. The command as you've included it does work. From my testing, looks like providing a filepath stops parsing the arguments after that point so if, for example, the command was organized as:

`$  PYTHONPATH=yapf python yapf/yapf /tmp/a.py --style='{based_on_style: pep8, column_limit: 99}' `

Then it would ignore the style options. Similarly for the others, such as -d and -r. Does that help?

**Comment by gwelymernans** [comment]
Yes. It was an "argparse" thing we were using.

Closed with 36465d0.
